src/modules/Inquiries.js

```diff
@@ -9,7 +9,7 @@ const Inquiries = {
     } catch (error) {}
   },
   async update(id, status, setInquiryStatus) {
-    let params = { form_data: { inquiry_status: status } }
+    let params = { form_data: { status_action: status } }
```

Comment:
we have made changes on the back end, instead of `form_data` you should use `inquiry`

Reply:
also make sure that you send off the event names to change the `inquiry_state` on the background instead of the desired `inquiry_state`

e.g
`set_to_pending` for started to pending
`start` for pending to started
`finish` for started to done

Reply:
The new values are fixed, was sure that we'd fix the wrapper also. Must've been on another branch :P